libminifi/include/Exception.h

```diff
@@ -60,16 +60,17 @@ class Exception : public std::exception {
  public:
   // Constructor
   /*!
-   * Create a new flow record
+   * Create a new exception
    */
-  Exception(ExceptionType type, const char *errorMsg)
+  Exception(ExceptionType type, std::string errorMsg)
       : _type(type),
-        _errorMsg(errorMsg) {
+        _errorMsg(std::move(errorMsg)) {
```

Comment:
Just saw this when I was typing the one below, I'll run the build before merging obviously, but does this not generate an elision warning?

Reply:
As far as I remember that warning is only applied for return values, where moving could prevent copy-elision, but it's not the case.

Reply:
Ah yah. I don't remember either, but that sounds right  If I don't see the warning when I run the build then all is well that ends well. Thanks.